src/components/DatePicker/CalendarHeader/__tests__/CalendarHeader.spec.tsx

```diff
@@ -22,42 +25,64 @@ describe('CalendarHeader', () => {
     });
 
     it('should render correctly', () => {
-        expect(toJson(wrapper)).toMatchSnapshot();
+        expect(view.container).toMatchSnapshot();
+        expect(screen.getByRole('presentation')).toBeInTheDocument();
+        expect(screen.getAllByRole('option')).toHaveLength(YEARS[1] - YEARS[0] + 1);
     });
+
     it('should log an error and render nothing if min year is larger then max year', () => {
         // don't log to test output
         const consoleErrorSpy = jest.spyOn(console, 'error').mockImplementationOnce(() => {});
-        wrapper.setProps({ yearsRange: [9, 2] });
-        wrapper.update();
+        view.rerender(
+            // @ts-ignore
+            <CalendarHeader
+                yearsRange={[9, 2]}
+                date={new Date('2021-12-31')}
+                decreaseMonth={decreaseMonth}
+                increaseMonth={increaseMonth}
+                changeYear={changeYear}
+            />
+        );
 
-        expect(wrapper.html()).toBe(null);
         expect(consoleErrorSpy).toHaveBeenCalledTimes(1);
         expect(consoleErrorSpy.mock.calls[0][0].replace(/\s/g, '')).toMatch(
```

Comment:
Do we need to replace the expected and the actual values the same thing? If we don't replace in both it should work no?

Reply:
Without it returns 
![image](https://user-images.githubusercontent.com/104063210/235893078-8cd284f1-4538-4cb1-a1b9-11185438865d.png)